recipes/lzma_sdk/9.20/test_package/conanfile.py

```diff
@@ -0,0 +1,8 @@
+from conans import ConanFile
+
+
+class TestPackage(ConanFile):
+
+    def test(self):
+        self.run("7zDec")
```

Comment:
This just tests that the executable is in the path? Does this only build executables and not libraries?

Reply:
There is a LZMA library (for Visual Studio only) built in `C/Util/LzmaLib` that exposes 2 symbols: `LzmaCompress` and `LzmaUncompress`.
I tried to package that library and write a test but but I couldn't recover the original data after a compress and decompress.
Also, because the library would only be available on Visual Studio, I decided to not package it.